### ISSUE TITLE
Feature request: easy reboot

### ISSUE THREAD

**Issue by sshaikh** [spocky/miproja1#136]
I currently have to access settings -> android settings -> reboot (set as a shortcut).

Some power options in the main dash would be great.

**Comment by crixyd** [comment]
Agreed, this would be very helpful. Mostly for the wife acceptance factor.

**Comment by engyrus** [comment]
I have a row of such "tools" on my home screen that I set up with TVQuickActions Pro. (you can publish any TVQA menu as a channel then display that channel in Projectivy.) to me it is best not to add all sorts of things to Projectivy when a very capable external app is available.

**Comment by sshaikh** [comment]
Thank for the tip. Tried to set it up but failed miserably, so I would still say there's value in in adding these basic features (which I don't consider "all sorts of things") into the core experience, even via the settings menu.
